examples/src/main/scala/sttp/client3/examples/RetryZio.scala

```diff
@@ -24,6 +26,6 @@ object RetryZio extends ZIOAppDefault {
 
         sendWithRetries.ensuring(backend.close().ignore)
       }
-      .provide(Clock.live)
+      .provide(ZLayer.succeed[Clock](ClockLive)(Tag[Clock], Tracer.newTrace))
```

Comment:
I think this line can go entirely?

Reply:
Unfortunately not at least as far as I tried, I think that `ZIOAppDefault` require `Clock` layer as after removing it I got compilation error

Reply:
OOkk my bad sorry, so `ZIO[Clock, Throwable, Response[String]]` to `ZIO[Any, Throwable, Response[String]]` ?

Reply:
Yeah, basically all `Clock` in the environment can be removed

Reply:
great thanks